versioned_docs/version-2.x/magento1/search-engine.mdx

```diff
@@ -584,9 +587,11 @@ Add a resolver for this query:
 ```js title="my-module/server/modules/SuggestedProducts/resolvers.js"
 export default {
   Query: {
-    productRecommendation: async (_, __, { loaders }) => {
+    productRecommendation: async (_, { profileId }, { loaders }) => {
       return loaders.AttraqtRecommendations.loadRecommendationsForWidget(
-        "my-widget-id"
+        "my-widget-id",
+        null,
+        profileId
```

Comment:
**issue:** we must mention that it's possible since 2.28. Otherwise developers may try to achieve this with a previous version.

**suggestion:** add a `<Since>` component, and maybe also an inline comment here

```suggestion
        profileId // available since 2.28
```